pkg/api/index.go

```diff
@@ -159,7 +159,15 @@ func (hs *HTTPServer) getNavTree(c *models.ReqContext, hasEditPerm bool) ([]*dto
 		{Text: "Divider", Divider: true, Id: "divider", HideFromTabs: true},
 		{Text: "Manage", Id: "manage-dashboards", Url: setting.AppSubUrl + "/dashboards", Icon: "sitemap"},
 		{Text: "Playlists", Id: "playlists", Url: setting.AppSubUrl + "/playlists", Icon: "presentation-play"},
-		{Text: "Snapshots", Id: "snapshots", Url: setting.AppSubUrl + "/dashboard/snapshots", Icon: "camera"},
+	}
+
+	if c.OrgRole != models.ROLE_VIEWER {
```

Comment:
this is better types has if c.HasRole(models.ROLE_EDITOR)   (will be true for admins as well)

Reply:
I think this check need to consider admin/editor permission in folders/dashboards and should use `hasEditPerm` argument instead. But, needs to be verified.

Reply:
@marefr since viewers cannot create playlist today, I think we can wait with this until we make it possible to create playlists inside a folder

Reply:
@torkelo yes that's true but this check is specifically for snapshots and not playlists, right? Platlist navitem is above.

Reply:
Ah , Sam your right . Mistook this item for playlists

Reply:
@torkelo @jackw need some changes to move this PR forward I suppose?